python/GafferDelight/__init__.py

```diff
@@ -38,4 +38,21 @@
 
 from ._GafferDelight import *
 
+# GafferDelight does not need any symbols from IECoreDelight, so MSVC tries
+# to be helpful and not load IECoreDelight.dll. Skipping that means that
+# 3Delight is never registered as a renderer, so we import / register it manually.
+
+import os
+
+ieCoreDelight = None
```

Comment:
This variable seems to be unused (other than to be assigned again at L54) - do we really need it?

Reply:
I also use it to on L49 to check that we haven't already imported `IECoreDelight.dll`. I think that was left over from a debugging effort though. I don't think there's much harm in asking for an already loaded library to be loaded.

Reply:
Is it possible for it to be set `True` between L47 and L49 though?

Reply:
Ahhhhhh, right. Don't mind me. 🗿 Fixed in https://github.com/GafferHQ/gaffer/pull/5082/commits/e95e2f39e5cb206f9514aebc172537c1240b5817